package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@asyncapi/parser",
-  "version": "0.33.1",
+  "version": "1.0.0-rc.1",
```

Comment:
Looks like you're missing a merge with master?

Reply:
no, rc releases go from my local, this PR is `bump to release candidate` 😉